.env.example

```
@@ -83,3 +83,7 @@ X_REDIRECT_URI="${APP_URL}/auth/x/callback"
 ADMIN_EMAIL=
 ANALYTICS_EXPORT_TOKEN=
 ANALYTICS_SNAPSHOT_TIMEZONE="${APP_TIMEZONE}"
+
+VAPID_SUBJECT="https://mydelight.app"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)